proto/osmosis/concentrated-liquidity/position.proto

```diff
@@ -58,8 +58,7 @@ message FullPositionBreakdown {
     (gogoproto.moretags) = "yaml:\"claimable_incentives\"",
     (gogoproto.nullable) = false
   ];
-  repeated cosmos.base.v1beta1.DecCoin forfeited_incentives = 6 [
-    (gogoproto.castrepeated) = "github.com/cosmos/cosmos-sdk/types.DecCoins",
+  repeated cosmos.base.v1beta1.Coin forfeited_incentives = 6 [
```

Comment:
Note to reviewer: returned all forfeited incentives back to Coins from DecCoins since we drop the dust now.

Reply:
I don't think I understand what was going on with dust here before. Not having dust in this seems right